share/src/main/java/org/alfresco/web/scripts/RssDashletStatusWebScript.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2005 - 2020 Alfresco Software Limited.
```

Comment:
The copyright year will need to be addressed as part of a separate task for all source files plus the license header text file used by the license check plugin.